README.md

```diff
@@ -71,6 +71,21 @@ Also included is a utility to validate a successful deployment.
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```

Comment:
`[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)`

Reply:
please test this